### PR TITLE
Fat 822/c3506

### PR DESCRIPTION
**Pull request by kimkate303**
https://foliotest.testrail.io/index.php?/cases/view/3506

### CODE REVIEW

File: cypress/support/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ import 'cypress-file-upload';
 import './data-import';
 import './commands';
 
-setInteractorTimeout(80_000);
+setInteractorTimeout(10_000);
```
Comment (MaksimTerekhov) [inline cypress/support/index.js:20]: Why you change it? As I know we a tests that need this 80000 value

File: cypress/support/fragments/orders/orders.js [hunk 2]
```diff
@@ -22,6 +22,13 @@ const buttonSubscriptionFromFilter = Button({ id: 'accordion-toggle-button-subsc
 const searchForm = SearchField({ id: 'input-record-search' });
 
 export default {
+  waitLoading() {
+    cy.expect([
+      Pane({ id: 'orders-filters-pane' }).exists(),
+      Pane({ id: 'orders-results-pane' }).exists(),
```
Comment (IhorBohdan) [inline cypress/support/fragments/orders/orders.js:28]: move to const Pane

File: cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import newOrder from '../../../support/fragments/orders/newOrder';
+import basicOrderLine from '../../../support/fragments/orders/basicOrderLine';
```
Comment (penfriend) [inline cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js:2]: import fragments from the UpperCase letter

File: cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+describe('orders: Receive piece from Order', () => {
+  const order = { ...newOrder.defaultOrder };
+  const orderLine = { ...basicOrderLine.defaultOrderLine };
+  let orderNumber;
+  const barcode = Helper.getRandomBarcode();
+  const caption = 'autotestCaption';
+
+  before(() => {
+    cy.getAdminToken();
+    cy.getOrganizationApi({ query: 'name="Amazon.com"' })
```
Comment (penfriend) [inline cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js:22]: Please move all values of query name parameter to variables. for example const organizationName = 'Amazon.com'

**Review by penfriend — CHANGES_REQUESTED** [review]

File: cypress/support/index.js (hunk 1 above)
Comment (kimkate303) [reply cypress/support/index.js]: yep, restore the previous version

File: cypress/support/fragments/orders/orders.js (hunk 2 above)
Comment (kimkate303) [reply cypress/support/fragments/orders/orders.js]: done

File: cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js (hunk 3 above)
Comment (kimkate303) [reply cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js]: done)

File: cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js (hunk 4 above)
Comment (kimkate303) [reply cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js]: done)

File: cypress/support/fragments/inventory/inventorySearch.js [hunk 5]
```diff
@@ -219,4 +219,8 @@ export default {
   verifySearchResult(cellContent) {
     cy.expect(MultiColumnListCell({ content: cellContent }).exists());
   },
+
+  expandHoldingsSection() {
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/inventory/inventorySearch.js:223]: this is excess method. Don't use it please.
use openHoldings from InventoryInstance instead

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
+import testType from '../../../support/dictionary/testTypes';
+
```
Comment (penfriend) [inline cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js:2]: Please remove this empty space.

File: cypress/support/fragments/inventory/inventorySearch.js (hunk 5 above)
Comment (kimkate303) [reply cypress/support/fragments/inventory/inventorySearch.js]: thank, done

**Review by penfriend — APPROVED** [review]

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**